packages/components/nodes/chatmodels/ChatOpenAI/ChatOpenAI.ts

```diff
@@ -152,6 +152,73 @@ class ChatOpenAI_ChatModels implements INode {
                 type: 'json',
                 optional: true,
                 additionalParams: true
+            },
+            {
+                label: 'Allow Image Uploads',
+                name: 'allowImageUploads',
+                type: 'boolean',
+                default: false,
+                optional: true
+            },
+            {
+                label: 'Allow Audio Uploads',
+                name: 'allowAudioUploads',
```

Comment:
why the need for both `allowSpeechToText` and `allowAudioUploads` ?

Reply:
speechToText to allow Microphone recording as input, audioUploads to upload existing mp3/wav/... to be used as input (similar to images)

Reply:
although its good to have more options, but I think its kinda confusing to end users. 99% of users prob just want something similar to alexa, voice assistant, which is the speech to text toggle only.